.github/CODEOWNERS

```diff
@@ -44,6 +44,7 @@
 # Documentation
 *.md  @oneapi-src/onednn-doc @oneapi-src/onednn-arch @intel-innersource/dnn-doc @intel-innersource/dnn-arch
 /doc/ @oneapi-src/onednn-doc @oneapi-src/onednn-arch @intel-innersource/dnn-doc @intel-innersource/dnn-arch
+/doc/graph/ @oneapi-src/onednn-doc @oneapi-src/onednn-arch @oneapi-src/onednn-graph @intel-innersource/dnn-doc @intel-innersource/dnn-arch @intel-innersource/dnn-graph
```

Comment:
This change would make approval from any member of `onednn-graph` to cover documentation part as well. See a note in [CODEOWNERS documentation](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-and-branch-protection). I would rather add you to `onednn-doc`.

Reply:
I see. Thanks Vadim. I dropped the change for codeowner but still want to keep the change for labeler. Could you please take another look?

Reply:
Looks good!